projects/igniteui-angular-elements/src/themes/light/bootstrap.scss

```diff
@@ -1,10 +1,20 @@
 /* You can add global styles to this file, and also import other style files */
 @use '../../../../igniteui-angular/src/lib/core/styles/themes' as igx;
 @use '../variables' as vars;
+@use '../util' as util;
 @import url('https://fonts.googleapis.com/icon?family=Material+Icons');
 
 $palette: igx.$light-bootstrap-palette;
 
 @include igx.core();
 @include igx.typography($font-family: igx.$bootstrap-typeface, $type-scale: igx.$bootstrap-type-scale);
-@include igx.bootstrap-light-theme($palette: $palette, $exclude: vars.$unused-themes);
```

Comment:
Only concern from moving away from using `bootstrap-light-theme` and so on - that mixin seems to do something extra, rather than using the `light-bootstrap-palette` directly for some reason:
https://github.com/IgniteUI/igniteui-angular/blob/91669a690fdd952201d59fbd9eefacf5f2014fce/projects/igniteui-angular/src/lib/core/styles/themes/generators/_bootstrap.scss#L33-L34
Kinda odd